control/control.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, CESAR. All rights reserved.
+#
+# SPDX-License-Identifier: BSD 3-Clause
+
+import dbus
+import dbus.service
+import dbus.mainloop.glib
+import os
+
+from reset import createReset
+from reset import Reset
```

Comment:
You are importing this and it is not using. Besides you won't need this class as the function of createReset is to import create an instance to you.